Puppetfile

```diff
@@ -10,3 +10,7 @@ mod 'jfryman/selinux'
 mod 'pcfens/rhn_register'
 mod 'stankevich/python'
 mod 'kemra102/cloudwatchlogs'
+
+# TODO: pull from github until it is pushed to puppet forge
```

Comment:
v0.9.0 has been published to Forge, please give it a try.
https://forge.puppet.com/shinesolutions/aem_resources